lib/prelude.mli

```diff
@@ -14,7 +14,9 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
 
-val env_and_file : string -> Block.Env.t * string
+val env_and_file : string -> Block.Env.t Util.One_or_all.t * string
```

Comment:
I understand `All | One of 'a` is unambiguous when it comes to envs but we could use an option here as well.

If we want to use this type we should probably move it to the `Env` module.

What do you think?

Reply:
I thought of using an option type but I thought it would be too confusing as we want to convey a different information